src/its_preselector/controlbyweb_web_relay.py

```diff
@@ -77,7 +81,15 @@ def get_digital_input_value(self, input_num: int) -> bool:
             digital_input = root.find(input_tag)
         if digital_input is None:
             raise ConfigurationException(f"Digital Input {input_num} does not exist.")
-        return bool(digital_input.text)
+        elif digital_input == "1":
```

Comment:
I think this needs to be digital_input.text == "1". Also. I think you could just do bool(int(digital_input.text))

Reply:
Yep, that's much better. Switched to that method in [7cbb844](https://github.com/NTIA/Preselector/pull/20/commits/7cbb844becadf57846e6f0a9f2c4aa4f5d738dc9). Tested and working on seadog01.